### PR TITLE
feat: add gam test

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: src/_includes/layouts/post.njk [hunk 1]
```diff
@@ -61,9 +61,13 @@ time #}
...
-                                {% for i in range(0, numOfSidebarAds) %}
-                                    {% include "partials/ad.njk" %}
-                                {% endfor %}
+                                {% if canonicalUrl == "https://www.freecodecamp.org/news/are-you-being-micro-managed-manage-your-relationship-with-your-manager-instead-9ad10b28bcda/" %}
+                                    {% include "partials/gam-ad.njk" %}
+                                {% else %}
+                                    {% for i in range(0, numOfSidebarAds) %}
+                                        {% include "partials/ad.njk" %}
+                                    {% endfor %}
+                                {% endif %}
```
Comment (scissorsneedfoodtoo) [inline src/_includes/layouts/post.njk:70]: Just to double check, is the goal for testing GAM with this one article just to add a single ad block to the sidebar for the target article?

Or should do something closer to the current method of adding ad blocks to the sidebar based on read time?

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Hey @ahmaxed, everything here looks good. When I swap out the target article for one in the Dockerized Ghost instance and build English locally using Dockerized Ghost, the expected scripts are loaded, and there's a single ad block in the sidebar only for the expected article.

I just have one quick question about the number of ads you want to test in the sidebar.

File: src/_includes/layouts/post.njk (hunk 1 above)
Comment (ahmaxed) [reply src/_includes/layouts/post.njk:70]: I am just testing to see the ad is working properly. The placement and lazy loading is in the works.

Comment (ahmaxed) [reply src/_includes/layouts/post.njk:70]: So only one ad for now. I want to make sure that it gets filled and displayed properly.

**Review by scissorsneedfoodtoo — APPROVED** [review]
Everything LGTM 👍 

Thanks again for the chat about this and donations, @ahmaxed :)

**Pull request merged**